Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM debian:9
+FROM debian:10
 
 RUN apt-get update && \
-    apt-get install -y build-essential wget libpq-dev libffi-dev python-dev postgresql-client && \
+    apt-get install -y build-essential python3-pip \
+      libpq-dev libffi-dev python3-dev postgresql-client && \
     rm -rf /var/lib/apt/lists/*
 
-RUN wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py && python /tmp/get-pip.py
 
-ADD ./requirements_base.txt /requirements/pip/base.txt
-RUN pip install --require-hashes -r /requirements/pip/base.txt
+COPY requirements_*.txt /tmp/
 
-ADD ./requirements_dev.txt /requirements/pip/dev.txt
-RUN pip install --require-hashes -r /requirements/pip/dev.txt
+RUN pip3 install --require-hashes -r /tmp/requirements_base.txt
+RUN pip3 install --require-hashes -r /tmp/requirements_dev.txt
+RUN pip3 install --require-hashes -r /tmp/requirements_tests.txt
```

Comment:
This should be folded into a single command:

```dockerfile
RUN pip3 install --require-hashes -r /tmp/requirements_base.txt -r /tmp/requirements_dev.txt -r /tmp/requirements_tests.txt
```

Reply:
Folded.